package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -182,9 +191,12 @@ export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProv
     }
   }, [renderStep, current]);
 
-  const goTo = useCallback((index: number): void => {
-    renderStep(index);
-  }, [renderStep]);
+  const goTo = useCallback(
+    (index: number): void => {
+      renderStep(index);
+    },
+    [renderStep],
+  );
```

Comment:
Are you using Prettier to reformat this file? We intentionally don't use prettier in the project because of a couple of reasons:
- It's too opinionated. It has no flexibility to choose how the code should be formatted. This is especially a problem with React code. Like in this example, we should preserve the format because this is a hook. It's more straightforward to read (it looks like a function, and we always have the dependencies below). More importantly, it's easy to spot as a React hook. Prettier moves the arguments to the next line; it doesn't care that this is intended as a hook, making things more complicated.
- Developers can do better: Prettier may be helpful in big teams where it takes more work to keep track of the format. Otherwise, we should strive to make the code better and cleaner, including knowing better than Prettier to make things more readable and maintainable.